api/v1alpha1/oadp_types.go

```diff
@@ -201,9 +205,9 @@ type DataMover struct {
 	// defines how often (in days) to prune the datamover snapshots from the repository
 	// +optional
 	PruneInterval string `json:"pruneInterval,omitempty"`
-	// defines configurations for data mover volume options
+	// defines configurations for data mover volume options for a storageClass
 	// +optional
-	DataMoverVolumeOptions *DataMoverVolumeOptions `json:"volumeOptions,omitempty"`
+	StorageClass map[string]DataMoverVolumeOptions `json:"storageClass,omitempty"`
```

Comment:
Are we just removing volumeoptions in general?
Thought we were looking to expose the useMoverContext elsewhere.

Reply:
This is a separate issue from securityContext. This is to create a mapping of storgeClasses to volumeOptions

Reply:
The name is confusing. Calling it `StorageClass` makes it look like the value is the name of a storage class rather than a mapping of storage classes to volume options. The original `DataMoverVolumeOptions` name might be better, or maybe `DataMoverVolumeOptionsForStorageClasses` (or shortened to `VolumeOptionsForStorageClasses`)